nurseconnect/urls.py

```diff
@@ -151,3 +150,9 @@
     urlpatterns += static(
         settings.MEDIA_URL + "images/",
         document_root=os.path.join(settings.MEDIA_ROOT, "images"))
```

Comment:
Can we not move that last closing bracket to the next line, to stick to a single convention?